src/components/InputField/index.js

```diff
@@ -17,7 +17,7 @@ InputField.propTypes = {
   id: PropTypes.string.isRequired,
   label: PropTypes.node,
   inputRef: PropTypes.shape({
-    current: PropTypes.instanceOf(Element),
+    current: typeof Element !== 'undefined' ? Element : Object,
```

Comment:
shouldn't it contain `PropTypes.instanceOf`?

Reply:
same for the other ones

Reply:
Yeah :facepalm: fixed!